aiohttp/_websocket/reader_py.py

```diff
@@ -45,9 +47,55 @@
 TUPLE_NEW = tuple.__new__
 
 
+class WebSocketDataQueue(DataQueue[WSMessage]):
+    """WebSocketDataQueue resumes and pauses an underlying stream.
+
+    It is a destination for WebSocket data.
+    """
+
+    def __init__(
+        self, protocol: BaseProtocol, limit: int, *, loop: asyncio.AbstractEventLoop
+    ) -> None:
+        super().__init__(loop=loop)
+        self._size = 0
+        self._protocol = protocol
+        self._limit = limit * 2
+
+    def feed_data(self, data: WSMessage) -> None:
+        payload = data[0]
```

Comment:
I think this is more readable (and robust against refactorings) to use the attribute. Seems to be the same performance-wise too.

```suggestion
    def feed_data(self, msg: WSMessage) -> None:
        payload = msg.data
```